cdap-ui/app/cdap/components/Home/HomeHeader/index.js

```diff
@@ -83,6 +92,52 @@ export default class HomeHeader extends Component {
       </Dropdown>
     );
 
+
+    const PaginationDropdown = () => {
+      let number = this.props.numberOfPages;
```

Comment:
When we introduce static type systems this could be a problem. Rename this to `numberOfPages` instead of just `number`